pyproject.toml

```diff
@@ -22,7 +22,7 @@ dependencies = [
     "munch>=2.5.0,<5.0",
     "numpy>=1.22.2,<2",
     "protobuf>=3.19.0,<5",
-    "py-to-proto>=0.5.0,<0.6.0,!=0.2.1",
+    "py-to-proto>=0.5.2,<0.6.0",
```

Comment:
Was something released recently to make us want to have this lower bound bump? Unless we NEED a feature in `0.5.2`, I think we should try to keep the range as open as possible.

Reply:
Yeah- we need 0.5.2 to support singular oneofs: https://github.com/IBM/py-to-proto/releases/tag/0.5.2